test/packages/parallel/oracle/data_stream/memory/_dev/test/system/test-memory-config.yml

```diff
@@ -0,0 +1,17 @@
+vars:
+  hosts:
+    - "oracle://sys:Oradoc_db1@elastic-package-service-oracle-1:1521/ORCLCDB.localdomain?sysdba=1"
+agent:
+  runtime: docker
+  custom_script: |
```

Comment:
Added for `custom_script` (now it's named `provisioning_script`) and also for `pre_start_script`. 

For `pre_start_script` , it is just allowed language `sh`, since it is injected directly those commands into the start script (entrypoint.sh).